assets/js/base/components/cart-checkout/form-step/index.tsx

```diff
@@ -27,6 +31,18 @@ const StepHeading = ( { title, stepHeadingContent } ) => (
 	</div>
 );
 
+interface FormStepProps {
+	id?: string;
+	className?: string;
+	title?: string;
+	legend?: string;
+	description?: string;
+	children?: JSX.Element | JSX.Element[] | string | string[];
```

Comment:
Does using `React.ReactNode` here work instead?

Reply:
It does indeed 👍